ui/src/actions/actions-ui.js

```diff
@@ -8,6 +8,7 @@ import {
   APPS_SORT,
   TOGGLE_HEADER_DROPDOWN,
   TOGGLE_NAVIGATION_MODAL,
+  TOGGLE_CONFIRM_SAVE_APP_MODAL,
```

Comment:
```suggestion
  TOGGLE_SAVE_APP_MODAL,
```

just wonder if we need the `CONFIRM`

Reply:
I have it as `CONFIRM` because the modal does not actually execute the logic to save the app, it gives the go ahead to the parent container to do this. So having it as `SAVE_APP_MODAL` could cause one to believe it will be saving the app. I can still change it if it is still unnecessary, but ant to hear your thoughts on my reason for naming it as this.

Reply:
Hmm, my own reading of it would be `TOGGLE_SAVE_APP_MODAL` (or `TOGGLE_CONFIRM_SAVE_APP_MODAL`) makes it clear enough it's just a toggling of the modal going on, so proposed rename would just be to reduce verbosity. Either name is good to me though

Reply:
Changed.